docs/core_docs/docs/integrations/vectorstores/libsql.mdx

```diff
@@ -0,0 +1,112 @@
+# libSQL
```

Comment:
Can we make the docs follow this template?

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-scripts/src/cli/docs/templates/vectorstores.ipynb

I need to update the contributing instructions...

Reply:
@jacoblee93 I updated the docs to follow more closely your example, however if you have time, please suggest or commit any changes you desire.